src/main/java/com/spotify/reaper/resources/ClusterResource.java

```diff
@@ -139,6 +142,16 @@ public Response addCluster(
     } else {
       LOG.info("creating new cluster based on given seed host: {}", newCluster.getName());
       context.storage.addCluster(newCluster);
+
+      if (context.config.hasAutoSchedulingEnabled()) {
+        try {
+          clusterRepairScheduler.scheduleRepairs(newCluster);
+        } catch (ReaperException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)